includes/templates/class-amp-content-sanitizer.php

```diff
@@ -88,7 +88,7 @@ public static function sanitize_document( Document $dom, $sanitizer_classes, $ar
 			 *
 			 * @type AMP_Base_Sanitizer $sanitizer
 			 */
-			$sanitizer = new $sanitizer_class( $dom, array_merge( $args, $sanitizer_args ) );
+			$sanitizer = new $sanitizer_class( $dom, array_merge_recursive( $sanitizer_args, $args ) );
```

Comment:
Used `array_merge_recursive` here so that the `element_xpaths` argument could be merged with the default sanitization arguments.

Reply:
This will probably need to be reverted if my suggestion above is implemented.